tox.ini

```
@@ -26,7 +26,7 @@ setenv =
     PYTHONDONTWRITEBYTECODE=1
 deps =
     -e .
-    py{3,310,311,312,313,py3}: -rtests/requirements/py3.txt
+    py{3,312,313,py3}: -rtests/requirements/py3.txt
```

```suggestion
    py{3,312,313}: -rtests/requirements/py3.txt
```

Should PyPy also be removed here?

---

Right, removed :+1: